api/app/Extensions/JWTAuth/JWTAuth.php

```diff
@@ -0,0 +1,58 @@
+<?php namespace App\Extensions\JWTAuth;
+
+use Illuminate\Http\Request;
+use Tymon\JWTAuth\Token;
+use Tymon\JWTAuth\JWTAuth as JWTAuthBase;
+use RuntimeException;
+use App\Exceptions\BadRequestException;
+use App\Exceptions\UnauthorizedException;
+use App\Exceptions\TokenInvalidException;
+use App\Exceptions\UnprocessableEntityException;
+use Tymon\JWTAuth\Exceptions\TokenExpiredException;
+
+class JWTAuth extends JWTAuthBase
+{
+    /**
+     * Get the token contained within the current request.
+     *
+     * @param  Request  $request
+     * @throws BadRequestException
+     * @return Token
+     */
+    public function getTokenFromRequest(Request $request)
```

Comment:
Actually, after further research, this only happens from unit testing. So will adjust the tests, and then update to use the constructor injected request.